### PR TITLE
[features test cmd] Respect image label metadata

### PR DESCRIPTION
**Pull request by joshspicer**
Adds a new flag `--skip-image-metadata` with a default state of `false`, reflecting the default behavior of the CLI to apply [container image label metadata](https://containers.dev/implementors/json_reference/) to the resultant image.

### CODE REVIEW

File: src/spec-node/featuresCLI/test.ts [hunk 1]
```diff
@@ -17,8 +17,9 @@ export function featuresTestOptions(y: Argv) {
 			'skip-scenarios': { type: 'boolean', default: false, description: 'Skip all \'scenario\' style tests.  Cannot be combined with \'--global--scenarios-only\'.' },
 			'skip-autogenerated': { type: 'boolean', default: false, description: 'Skip all \'autogenerated\' style tests.' },
 			'base-image': { type: 'string', alias: 'i', default: 'ubuntu:focal', description: 'Base Image. Not used for scenarios.' },  // TODO: Optionally replace 'scenario' configs with this value?
-			'remote-user': { type: 'string', alias: 'u', default: 'root', describe: 'Remote user.  Not used for scenarios.', },  // TODO: Optionally replace 'scenario' configs with this value?
+			'remote-user': { type: 'string', alias: 'u', describe: 'Remote user.  Not used for scenarios.', },  // TODO: Optionally replace 'scenario' configs with this value?
 			'log-level': { choices: ['info' as 'info', 'debug' as 'debug', 'trace' as 'trace'], default: 'info' as 'info', description: 'Log level.' },
+			'apply-image-dev-container-metadata': { type: 'boolean', default: true, description: 'Apply image metadata to the resultant image appended by previously building with the devcontainer CLI' },
```
Comment (chrmarti) [inline src/spec-node/featuresCLI/test.ts:22]: Would `--skip-image-metadata` align better with the other `skip` options above?

**Review by chrmarti — CHANGES_REQUESTED** [review]
LGTM, left one suggestion.

File: src/spec-node/featuresCLI/test.ts (hunk 1 above)
Comment (joshspicer) [reply src/spec-node/featuresCLI/test.ts:22]: Good point, I like that better.

**Review by samruddhikhandale — APPROVED** [review]
🚀 🚀

**Pull request merged**